website/versioned_docs/version-26.4/Configuration.md

```diff
@@ -4,7 +4,7 @@ title: Configuring Jest
 original_id: configuration
 ---
 
-Jest's configuration can be defined in the `package.json` file of your project, or through a `jest.config.js` file or through the `--config <path/to/file.js|cjs|mjs|json>` option. If you'd like to use your `package.json` to store Jest's config, the `"jest"` key should be used on the top level so Jest will know how to find your settings:
+Jest's configuration can be defined in the `package.json` file of your project, or through a `jest.config.js`, or `jest.config.ts` file or through the `--config <path/to/file.js|ts|cjs|mjs|json>` option. If you'd like to use your `package.json` to store Jest's config, the `"jest"` key should be used on the top level so Jest will know how to find your settings:
```

Comment:
changes in this file should be rolled back as `26.4` won't include support for this. Changing in `docs/` is correct, and I'll make sure to roll a new version of the docs when this is released